webapi/src/main/scala/org/knora/webapi/slice/ontology/domain/model/Cardinality.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.slice.ontology.domain.model
+import dsp.schema.domain.Cardinality.MayHaveMany
+import dsp.schema.domain.Cardinality.MayHaveOne
+import dsp.schema.domain.Cardinality.MustHaveOne
+import dsp.schema.domain.Cardinality.MustHaveSome
+import dsp.schema.domain.{Cardinality => OldCardinality}
+import org.knora.webapi.messages.v2.responder.ontologymessages.OwlCardinality.KnoraCardinalityInfo
+
+/**
+ * Represents a cardinality value object.
+ */
+sealed trait Cardinality {
+  def min: Int
+  def max: Option[Int]
+
+  val oldCardinality: OldCardinality
```

Comment:
Why is this needed?

Reply:
For mapping back and forth between both implementations.

Reply:
Yes, we currently have three `Cardinalities` and I would like to keep only the `org.knora.webapi.slice.ontology.domain.model.Cardinality`